pkgs/development/libraries/sqlite/tools.nix

```diff
@@ -4,11 +4,11 @@ let
   archiveVersion = import ./archive-version.nix lib;
   mkTool = { pname, makeTarget, description, homepage }: stdenv.mkDerivation rec {
     inherit pname;
-    version = "3.35.2";
+    version = "3.35.5";
 
     src = assert version == sqlite.version; fetchurl {
       url = "https://sqlite.org/2021/sqlite-src-${archiveVersion version}.zip";
-      sha256 = "00w7qggrdxpcrs4rfpqkjhcghi8pcx6zzjim9wlz6vsrxlr9dwva";
+      sha256 = "9StypcMZw+UW7XqS4SMTmm6Hrwii3EPXdXck9hMubbA=";
```

Comment:
```
nix-repl> :b sqlite-analyzer
hash mismatch in fixed-output derivation '/nix/store/q4z3cxqpvs9y98hkj9p1d9ggznwnwrzz-sqlite-src-3350500.zip':
  wanted: sha256:1c3d5q9zc93pfpbl7p5212pqfvls2ciy34ksxlbfbhqrqfjp4azm
  got:    sha256:049vdpk50sba786345ibmlxnkzk5zp4xj859658ancswb6jyrgpl
cannot build derivation '/nix/store/kwn8inpam22864hlip5b8cjjdv642md2-sqlite-analyzer-3.35.5.drv': 1 dependencies couldn't be built
```
Also on Hydra, e.g. https://hydra.nixos.org/build/145040451

Reply:
Ah, the hash is simply copied from `./default.nix` but that's wrong, as a different archive is used here.  How do we get to backport changes of such low quality?

Reply:
Fixed in 39889643872.

Reply:
Cross-ref: PR #124242.